cloudify/workflows/local.py

```diff
@@ -622,6 +694,30 @@ def get_capability(self):
         raise NotImplementedError('get_capability is not implemented by '
                                   'memory storage')
 
+    def get_executions(self):
+        try:
+            with open(self._executions_path) as f:
+                return json.load(f, object_hook=load_datetime)
+        except (IOError, ValueError):
+            return []
+
+    def store_executions(self, executions):
+        with open(self._executions_path, 'w') as f:
+            json.dump(executions, f, indent=2, cls=JSONEncoderWithDatetime)
+
+
+class JSONEncoderWithDatetime(json.JSONEncoder):
+    def default(self, obj):
+        if isinstance(obj, datetime):
+            return {'__datetime__': time.mktime(obj.timetuple())}
+        return super(JSONEncoderWithDatetime, self).default(obj)
+
+
+def load_datetime(dct):
```

Comment:
eh, it is just a dict that yet has to be examined for if it contains a datetime; `dct` is what the python docs use for an example of json's object_hook so i think i'll leave it as this

Reply:
I think those docs probably want a better name in their example? I feel like even something like 'data' is better than dct since dct sounds like it has some other meaning.